source/templates.tex

```diff
@@ -9,7 +9,7 @@
 \indextext{type generator|see{template}}
 
 \pnum
-A \term{template} defines a family of classes or functions or an alias for a
+A \term{template} defines a family of classes, functions or variables, or an alias for a
```

Comment:
Please use the Oxford comma here: "a family of classes, functions, or variables, or an alias for a family of types." Right now, this can be misread as "defines (a family of classes), (functions or variables), or (an alias for a family of types)."